<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAvatar</class>
 <widget class="QMainWindow" name="CreateAvatar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Avatar creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="body_type_label">
       <property name="font">
        <font>
         <family>Bernard MT Condensed</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Body Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SexComboBox">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="race_label">
       <property name="font">
        <font>
         <family>Bernard MT Condensed</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Race</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HSlid">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hair_label">
       <property name="font">
        <font>
         <family>Bernard MT Condensed</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Hair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HSlid1">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;</string>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="eyes_label">
       <property name="font">
        <font>
         <family>Bernard MT Condensed</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Eyes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HSlid3">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;</string>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rot_label">
       <property name="font">
        <font>
         <family>Bernard MT Condensed</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Rot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HSlid4">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;</string>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_backg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>424</width>
      <height>331</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/BackRes/res/backg_taverna.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bernard MT Condensed&quot;;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>15</y>
      <width>203</width>
      <height>293</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/BackRes/res/frame.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <zorder>graphicsView_backg</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton_Save</zorder>
   <zorder>graphicsView_frame</zorder>
   <zorder>graphicsView</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
